style changes for install page

Spreadyy opened the pull request.

The-Compiler commented:
@meles5 can you review this?

Spreadyy commented on www/qute.css:
```diff
@@ -136,6 +142,29 @@ code {
+
+.admonitionblock .title {
+	color: #0A396E;
+	font-weight: bold;
```
fixed font size stuff

meles5 commented:
The new table style is a great improvement! I like the way how it looks :)

But I think, a few rendered sites that have a footnote at the bottom of the page, like README.html and CONTRIBUTING.html, got one that doesn't fit in the new layout.

Furthermore on every rendered page there is a white line on top of the header. That's a `<p>` element, so you can just add a rule for that specific element to set its margin to zero.

In the table you wanted to display as a block, every code box got a grey line on its left hand side, just like the whole table, but the other code blocks outside don't have this line, so it would be nice if you could make these lines in the boxes disappear, too.

And the very last thing I'd like to mention is that on big screens the space around the content in the middle of the site might be too big.

But that's just my opinion, finally @The-Compiler will have to decide :)

Spreadyy commented:
@meles5 Thanks for the feedback.
Based on it I:
- Fixed footnotes
- Removed top line
- Fixed the code blocks inside the new table
- made the page 20% wider (1200px instead of 1000px)
Could you take a second look at it so @The-Compiler is able to merge the request? :D

meles5 commented:
Hi,

looks good, most of the things I've mentioned before are completely fixed now. But there's still a smaller problem with the footnotes:

The footnotes are now displayed correctly on larger screens, but not on smaller ones (at about 1280px in width and lower) -
the `<hr>` doesn't fit in with the layout again, it's longer than the main text (but the footnote-text is still fine).

By the way, I like the idea of the margin you've added to the `<hr>`! :)
But I think it might look better if you'd add some space under the footnotes itself and (optionally) set a lower margin beneath the `<hr>`.

Spreadyy commented:
hey @meles5 
I fixed the hr element on small devices and gave the footnotes a bit more padding.

meles5 commented:
Looks good! I think now everything should be ready to get merged by @The-Compiler 😉

The-Compiler commented:
Thanks @Spreadyy, and thanks @meles5 for the reviews!
